editor/src/components/canvas/controls/select-mode/subdued-grid-gap-controls.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { useColorTheme } from '../../../../uuiui'
+import { Substores, useEditorState, useRefEditorState } from '../../../editor/store/store-hook'
+import { useBoundingBox } from '../bounding-box-hooks'
+import { CanvasOffsetWrapper } from '../canvas-offset-wrapper'
+import type { Axis } from '../../gap-utils'
+import { gridGapControlBoundsFromMetadata, maybeGridGapData } from '../../gap-utils'
+import type { ElementPath } from 'utopia-shared/src/types'
+import { useGridData } from '../grid-controls'
+import { fallbackEmptyValue } from './controls-common'
+import type { CanvasRectangle } from '../../../../core/shared/math-utils'
+import type { CSSNumber } from '../../../../components/inspector/common/css-utils'
+
+export interface SubduedGridGapControlProps {
+  hoveredOrFocused: 'hovered' | 'focused'
+  axis: Axis | 'both'
+}
+
+export const SubduedGridGapControl = React.memo<SubduedGridGapControlProps>((props) => {
+  const { hoveredOrFocused, axis } = props
+  const targets = useEditorState(
+    Substores.selectedViews,
+    (store) => store.editor.selectedViews,
+    'SubduedGridGapControl selectedViews',
+  )
+  const metadata = useRefEditorState((store) => store.editor.jsxMetadata)
+  const selectedElement = targets[0]
+
+  const gridRowColumnInfo = useGridData([selectedElement])
+
+  const gridGap = maybeGridGapData(metadata.current, selectedElement)
+  if (gridGap == null) {
+    return null
+  }
+
+  const gridGapRow = gridGap.row
+  const gridGapColumn = gridGap.column
+
+  const controlBounds = gridGapControlBoundsFromMetadata(
+    selectedElement,
+    gridRowColumnInfo[0],
```

Comment:
Unsafe array indexing, it would appear in this case that it would break `gridGapControlBoundsFromMetadata` as that doesn't allow undefined to be passed to this parameter:
```suggestion
    gridRowColumnInfo.at(0),
```